src/test/java/io/strimzi/kafka/access/model/StatusUtilsTest.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.kafka.access.model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import io.strimzi.api.kafka.model.status.Condition;
+import io.strimzi.api.kafka.model.status.ConditionBuilder;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class StatusUtilsTest {
+    @Test
+    void testConditionUpdate() {
+        var c1 = new ConditionBuilder()
+            .withType(StatusUtils.CONDITION_TYPE_READY)
+            .withStatus(StatusUtils.CONDITION_STATUS_FALSE)
+            .withMessage("foo-1")
+            .withReason("bar")
+            .build();
+        var c2 = new ConditionBuilder()
+            .withType(StatusUtils.CONDITION_TYPE_READY)
+            .withStatus(StatusUtils.CONDITION_STATUS_FALSE)
+            .withMessage("foo-2")
+            .withReason("bar")
+            .build();
+        var c3 = new ConditionBuilder()
+            .withType(StatusUtils.CONDITION_TYPE_READY)
+            .withStatus(StatusUtils.CONDITION_STATUS_TRUE)
+            .withMessage("foo-3")
+            .withReason("bar")
+            .build();
+        var c4 = new ConditionBuilder()
+            .withType(StatusUtils.CONDITION_TYPE_READY)
+            .withStatus(StatusUtils.CONDITION_STATUS_TRUE)
+            .withMessage("foo-4")
+            .withReason("bar")
+            .build();
+
+        List<Condition> conditions = new ArrayList<>();
+        conditions.add(c1);
+
+        {
```

Comment:
I'd prefer if these test cases were broken out into separate, smaller test cases with an `@DisplayName` annotation explaining the aim of the test. That way they will be easier to refactor in future if the StatusUtils class is updated and if something breaks the test it's quicker to see exactly which behaviour is broken

Reply:
done.

let me know if it matches the expectation